guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,705 @@
 {
```

Comment:
<h4><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html" target="_blank">arcgis.raster.functions.gbl</a></code> </h4><ul><li>Adds new functions:</li><li class="ql-indent-1"><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html#boundary-clean" target="_blank">boundary_clean()</a> (Function available in</code>ArcGIS Image Server 10.9 or higher<code>)</code></li><li><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html#arcgis.raster.functions.gbl.zonal_statistics" target="_blank">zonal_statistics()</a></code></li><li class="ql-indent-1">Adds new parameter:</li><li class="ql-indent-1"><code>percentile_interpolation_type (Parameter available in</code>ArcGIS Image Server 10.9 or higher<code>)</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/820/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Note Added!